scripts/serve.mjs

```diff
@@ -70,6 +70,14 @@ http
           fontSize: 1.5,
           name: "PRICE (EUR)"
         },
+        xGrid: {
```

Comment:
It looks like #31 contains the patch of #30. If possible, create two independent PRs. Otherwise, if we merge #31, we automatically merge #30.